generators/server/templates/src/main/java/package/service/UserService.java.ejs

```diff
@@ -847,12 +847,16 @@ public class UserService {
             // Here we can't cater for all the use-cases, see:
             // https://github.com/jhipster/generator-jhipster/issues/7866
             // for the en_US or en_** issue.
-            // Since JHipter only have one "en" language, we will handle it here, for other languages, please handle it accordingly.
+            // Since JHipter only has one "en" language, we will handle it here,
+            // for other languages, please handle it accordingly.
             String locale = (String) details.get("locale");
-            if(locale.startsWith("en_") || locale.startsWith("en-")) {
-                locale = "en";
+            if (locale.startsWith("en_") || locale.startsWith("en-")) {
+                locale = Constants.DEFAULT_LANGUAGE;
```

Comment:
Shouldn't this be `en` to comply with if statement?

Reply:
Yep. Good catch. Fixed.